source/acquaman/IDEAS/IDEASXASScanActionController.cpp

```diff
@@ -217,18 +219,16 @@ AMAction3* IDEASXASScanActionController::createInitializationActions()
 void IDEASXASScanActionController::onInitializationActionsListSucceeded()
 {
 	AMControlInfoList positions(IDEASBeamline::ideas()->exposedControls()->toInfoList());
-	positions.remove(positions.indexOf("masterDwell"));
 	positions.remove(positions.indexOf("DirectEnergy"));
 	positions.remove(positions.indexOf("Energy"));
 	positions.remove(positions.indexOf("XRF1E Real Time"));
-
-	if(!configuration_->fluorescenceDetector().testFlag(IDEAS::NoXRF)){
-
-		positions.remove(positions.indexOf("XRF1E Peaking Time"));
-		positions.remove(positions.indexOf("XRF1E Trigger Level"));
-		positions.remove(positions.indexOf("XRF1E Baseline Threshold"));
-		positions.remove(positions.indexOf("XRF1E Preamp Gain"));
-	}
+	positions.remove(positions.indexOf("13-element Ge dwell time"));
```

Comment:
Not really.  It just means that those settings are being recorded for scans that aren't using that detector, which isn't an issue.